### PR TITLE
Fix event naming

### PR DESCRIPTION
**Pull request by tedtate**
Reviewers: @donnielrt @ericmuyser @fractaltheory 

Updates event naming style.

### CODE REVIEW

**Comment by fractaltheory** [comment]
:+1:

**Comment by ericmuyser** [comment]
Doesn't match the readme/docs? Perhaps I'm missing something, for consistency sake, I'm looking for where else are we using this opening/open (not opened) closing/close event convention in other plugins?
 
readme

**Comment by fractaltheory** [comment]
I think we'd just have to update the readme? If the intent of the PR was to fix a problem with the dots in the event names, then we'd want to update the readme to reflect that change, and then merge? Do you know if you had made this change because the dots weren't working?

**Comment by tedtate** [comment]
As far as I can tell the docs are just incorrect. I would like to get the following outcomes from this PR:

1) Make sure the names of the events in the docs are correct.
2) Verify the the dot style of event is working and fix if it isn't.

**Comment by ericmuyser** [comment]
OK sounds good. Updated the readme. Still not sure about opening/open but it's better than what's there. Two more quirks:

1) The point where it invokes magnifik:close isn't 100% legit either. It fires before any animations have finished.

2) Looking at this I'm skeptical it works in the case the image is loaded from cache: `this.$thumb.one('load', function() {`, so it won't properly center the image or fire magnifik:open.

My primary feeling is this plugin could use a rewrite with modern practices and the PR process.

File: README.md [hunk 1]
```diff
@@ -116,10 +116,10 @@ The magnifik emits the following events:
 
 | Name          | Description                               |   
 |---------------|-------------------------------------------|
-| beforeOpen    | Fired before magnifik starts opening      |
-| afterOpen     | Fired when magnifik is fully open         |
-| beforeClose   | Fired before magnifik starts closing      |
-| afterClose    | Fired after magnifik finishes closing     |
+| opening       | Fired before magnifik starts opening      |
```
Comment (donnielrt) [inline README.md]: Aren't the events magnifik:*? magnifik:closing, magnifik:opening, etc.

**Comment by donnielrt** [comment]
I think it's fine to include this bugfix PR to fix the addressed issue, but we can hold off any additional fixes for the next release.

**Comment by ericmuyser** [comment]
I suppose so @donnielrt :grinning: Updated!

**Comment by hcstephencheung** [comment]
Added examples folder. Also added console log statements to verify event names are indeed the same ones as README

**Pull request merged**